client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidCheckBoxSeekBarTest.java

```diff
@@ -40,22 +40,22 @@
     private static final String VALUE_TEXTVIEW_LOCATOR = "com.paypal.selion.pageobjectsdemoapp:id/stateTxtSwitch";
 
     private UiButton menuButton;
-    private UiObject seekBar;
+    private UiSlider seekBar;
     private UiTextView seekBarTextView;
     private UiTextView checkBoxTextView;
     private UiObject checkBox;
 
     @BeforeClass
     public void initElements() {
         menuButton = new UiButton(ACTION_BUTTON_LOCATOR);
-        seekBar = new UiObject(SEEK_BAR_LOCATOR);
+        seekBar = new UiSlider(SEEK_BAR_LOCATOR);
         seekBarTextView = new UiTextView(TEXT_VIEW_LOCATOR);
         checkBox = new UiObject(CHECKBOX_LOCATOR);
         checkBoxTextView = new UiTextView(VALUE_TEXTVIEW_LOCATOR);
     }
 
-    @Test
-    @MobileTest(appPath = PAGE_OBJECTS_APP_PATH, device = DEVICE_NAME)
+//    @Test
```

Comment:
I comment tests to run only one test and see the result, it might be commented out by mistake, the test is passed successfully though. Should it be commented again ?